WordPressKit/SiteDesignServiceRemote.swift

```diff
@@ -4,6 +4,7 @@ import WordPressShared
 public struct SiteDesignRequest {
     public enum TemplateGroup: String {
         case stable
+        case beta
         case singlePage = "single-page"
```

Comment:
Depending on [the outcome of this conversation](https://github.com/wordpress-mobile/WordPress-iOS/pull/18683#discussion_r877831031) we may remove the `singe-page` group